lib/core/janitor.js

```diff
@@ -374,7 +375,7 @@ class Janitor {
         body: objects[_id],
         controller: 'security',
         refresh: 'wait_for'
-      });
+      }, {user: { _id: 'User' } });
```

Comment:
"User" is not a valid kuid. You should revert that change and instead let the new getter I propose above handle the null user object.